diff --git a/dist/engine.js b/dist/engine.js
index f2695c500b48f14fc91590dfa9575a933738e647..d8ac26f5dd91cfb441534d44bc9bce2a5ca758ee 100644
--- a/dist/engine.js
+++ b/dist/engine.js
@@ -8,7 +8,7 @@ import { decodeEntry, encodeEntry, wrapEntry } from './journal.js';
 let sqlPromise = null;
 async function getSql() {
     if (!sqlPromise) {
-        sqlPromise = initSqlJs();
+        sqlPromise = initSqlJs({ locateFile: () => path.join(process.cwd(), "server/assets/sql-wasm.wasm") });
     }
     return sqlPromise;
 }
